Week10NetworkingCodingExercise/Week10NetworkingCodingExerciseTest/UnitTest1.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Net;
+using System.IO;
+using StreamsIONetworking;
+
+namespace Week10NetworkingCodingExerciseTest
+{
+    /// <summary>
+    /// Tests the code that is in the Program class.
+    /// </summary>
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
+        public void TestMethod1()
```

Comment:
I've updated this method to demonstrate how to test the web server using a WebClient. (-1.0)